### PR TITLE
New folder in rpcdqm online to test new readouts

### PR DESCRIPTION
**Pull request by slezki**
Adding new folder in RPC DQM to test new readout. And it is compatible with PR #24613 created by @ftorresd . @mileva @jhgoh @BrieucF @sandrofonseca

### CODE REVIEW

**Comment by andrius-k** [comment]
please test

**Comment by andrius-k** [comment]
please test

**Comment by slezki** [comment]
It seems that test has been finished as success in details, yesterday, but it  is still "Some checks haven’t completed yet". Is there any problem or is something running on the background (curiosity)?

File: DQM/Integration/python/clients/l1tstage2emulator_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -135,9 +135,7 @@
     process.gctDigis.inputLabel = cms.InputTag("rawDataRepacker")
     process.gtDigis.DaqGtInputTag = cms.InputTag("rawDataRepacker")
     process.twinMuxStage2Digis.DTTM7_FED_Source = cms.InputTag("rawDataRepacker")
-    process.RPCTwinMuxRawToDigi.inputTag = cms.InputTag("rawDataRepacker")
     process.bmtfDigis.InputLabel = cms.InputTag("rawDataRepacker")
-    process.valBmtfAlgoSel.feds = cms.InputTag("rawDataRepacker")
```
Comment (thomreis) [inline DQM/Integration/python/clients/l1tstage2emulator_dqm_sourceclient-live_cfg.py]: This should not have been removed.

Comment (slezki) [reply DQM/Integration/python/clients/l1tstage2emulator_dqm_sourceclient-live_cfg.py]: It is fixed.

**Comment by andrius-k** [comment]
please test

**Comment by andrius-k** [comment]
+1

**Comment by fabiocos** [comment]
+1

**Pull request merged**